src/fxos/index.js

```diff
@@ -0,0 +1,13 @@
+let modularize = require('../modularize');
+
+let Fxos = modularize(function Fxos(adb) {
```

Comment:
Sure, that way can work as well, it just means either instantiating an `Fxos` from the request method, or injecting `Adb` into the method as a parameter like you said. So either:

```js
// restart.js
let Fxos = require('./fxos');

module.exports = async function restart(req, res) {
  let fxos = new Fxos(req.adb);
  let hard = req.query.hard === 'true' || req.query.hard === '1';
  await fxos.reboot(hard);
  res.send('200 OK');
};
```

Or:

```js
// restart.js
let fxos = require('./fxos');

module.exports = async function restart(req, res) {
  let hard = req.query.hard === 'true' || req.query.hard === '1';
  await fxos.reboot(adb, hard);
  res.send('200 OK');
};
```